tests/test_il_corrections.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime
+from os.path import dirname, join
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.il_corrections import IlCorrectionsSpider
+
+test_pdf_response = file_response(
+    join(dirname(__file__), "files", "il_corrections.pdf"),
+    url="https://www2.illinois.gov/idoc/aboutus/advisoryboard/Documents/"
+    "Agenda%20-%20November%204th-2019.pdf",
+    mode="rb",
+)
+
+print(test_pdf_response)
+
+test_response = file_response(
+    join(dirname(__file__), "files", "il_corrections.html"),
+    url="https://www2.illinois.gov/idoc/aboutus/advisoryboard/Pages/default.aspx",
+)
+
+spider = IlCorrectionsSpider()
+
+freezer = freeze_time("2020-12-08")
+freezer.start()
+
+parsed_dates = spider._parse_all_links(test_response)
+test_generator = spider._meeting(test_pdf_response, "November 4, 2019")
+test_meeting = next(test_generator)
+
+freezer.stop()
+
+
+def meeting_count(parsed_dates):
```

Comment:
I think the test runner might not treat functions not starting in `test_` as tests, so this can become `test_meeting_count`